react/ui-components/src/atoms/stories/AlertPopup.stories.js

```
@@ -0,0 +1,320 @@
+  showAlertAsSvg: false,
+  equalWidthButtons: false,
+};
+
+const footerChildrenWithTwoButtons = [
+  <Button
+    type={"button"}
+    size={"large"}
+    variation={"secondary"}
+    label="Button"
+    onClick={() => console.log("Clicked Button 1")}
+  />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)